pkg/juicefs/juicefs.go

```diff
@@ -112,14 +116,24 @@ func (j *juicefs) IsNotMountPoint(dir string) (bool, error) {
 
 // JfsMount auths and mounts juicefs
 func (j *juicefs) JfsMount(volumeID string, secrets map[string]string, options []string) (Jfs, error) {
-	j.Upgrade()
-	stdoutStderr, err := j.AuthFs(secrets)
-	klog.V(5).Infof("MountFs: authentication output is '%s'\n", stdoutStderr)
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, "Could not auth juicefs: %v", err)
+	source, ok := secrets["metaurl"]
+	if !ok {
+		j.Upgrade()
+		stdoutStderr, err := j.AuthFs(secrets)
+		klog.V(5).Infof("MountFs: authentication output is '%s'\n", stdoutStderr)
+		if err != nil {
+			return nil, status.Errorf(codes.Internal, "Could not auth juicefs: %v", err)
+		}
+		source = secrets["name"]
```

Comment:
What is the reason for renaming `name` to `source` while keeping secret key not unchanged?

Reply:
Do we need to change the key name in secret from `name` to `source` for consistency? Or making an alias or something to make it self explainable.

Reply:
Good idea. Maybe we also need move some secret out from secrets to storage class parameters like `bucket`, `storage` etc.

Reply:
It's better to use `source` here, which could be redis-url or name of volume in juicefs.com.